Dockerfile

```diff
@@ -67,7 +68,6 @@ RUN set -ex \
     && pip install -U pip setuptools wheel \
     && pip install Cython \
     && pip install kubernetes \
-    && pip install cryptography \
```

Comment:
is this not needed ? I thought k8s needed this. Will add tests for all executors. For now retain this.

Reply:
you were right, i've just checked the requirements of `airflow` here: https://github.com/apache/incubator-airflow/blob/master/setup.py#L207